config/config-istio.yaml

```diff
@@ -45,6 +45,5 @@ data:
 
     # A cluster local gateway to allow pods outside of the mesh to access
     # Services and Routes not exposing through an ingress.  If the users
-    # do have a service mesh setup, this isn't required.
+    # do have a service mesh setup, this isn't required and can be removed.
```

Comment:
Include a comment about this being needed for our `istio-lean.yaml` setup?

Reply:
Done.